Updates to Episode 6 from Final Edits (#53)
datacarpentry/astronomy-python: _episodes/06-photo.md

The x-axis shows the difference in apparent magnitude between the g
and i filters, which indicates color.

Stars with lower values of (g-i) are "bluer" since they are brighter in the g-band than in

> the i-band, compared to other stars. To first order, the color of a star is 

Reviewer: ```suggestion
the i-band, compared to other stars. To a first order approximation, the color of a star is 
```

> related the star's temperature with bluer stars indicating higher temperatures and redder stars 

Reviewer: ```suggestion
related to the star's temperature, with bluer stars indicating higher temperatures and redder stars 
```

indicating lower temperatures. An important second order effect involves 

> the metallicity, or the amount of metals (elements heavier than He, in this case) 

Reviewer: ```suggestion
the metallicity, or the amount of metals (elements heavier than helium, in this case) 
```

that are present in a star's atmosphere. 
Higher metallicity leads to redder stars and lower metallicity leads to bluer stars. 

Stars in the lower-left quadrant of this diagram are faintest and bluest suggesting 

> they are have lower metallicity  

Reviewer: ```suggestion
they are have lower metallicity than
```

the other stars, which means they are [likely
to be
older](http://spiff.rit.edu/classes/ladder/lectures/ordinary_stars/ordinary.html).

> Since we expect the stars in GD-1 to be older than foreground and background

Reviewer: ```suggestion
Since we expect the stars in GD-1 to be older than the foreground and background
```

> stars, and far, the stars in the lower-left are more likely to be in GD-1.

Reviewer: ```suggestion
stars, and farther away, the stars in the lower-left are more likely to be in GD-1.
```

![Color magnitude diagram of our selected stars showing all of the stars selected](../fig/06-photo_files/06-cmd_no_lims.png)

In the previous cell we have assigned the color and magnitude to variables `x` and `y`, 
respectively.  

> We've done this out of convenience and to keep the code readable since the 

Reviewer: ```suggestion
We have done this out of convenience and to keep the code readable since the 
```
